cmd/deployment-check/deployment-check.yaml

```diff
@@ -10,15 +10,15 @@ spec:
   podSpec:
     containers:
     - name: deployment
-      image: kuberhealthy/deployment-check:v1.7.1
+      image: kuberhealthy/deployment-check:v1.8.0
       imagePullPolicy: IfNotPresent
       env:
         - name: CHECK_DEPLOYMENT_REPLICAS
           value: "4"
         - name: CHECK_DEPLOYMENT_ROLLING_UPDATE
           value: "true"
         - name: TOLERATION_VALUE
-          value: "dedicated"
+          value: "dedicated=kuberhealthy"
```

Comment:
These shouldn't be included in the example yaml because not everyone user will have these values, taints, and selectors

Reply:
I will remove these for my final push then. However I need these left in for testing purposes as my spec file I apply